partner_report_open_invoices/__openerp__.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# © 2016 Serpent Consulting Services Pvt. Ltd
```

Comment:
Copyright should be written in full, please see https://github.com/OCA/maintainer-tools/blob/master/template/module/models/__init__.py#L2 as reference.